site/content/api/Command-And-Control-API.md

```diff
@@ -12,43 +12,39 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
+3. *Business Application* has established an AMQP link in role *sender* (also using target address `control/${tenant_id}/${device_id}`). 
+
+**Link establishment and Message Flow**
 
-**Message Flow**
+The following sequence diagram shows the whole message exchange, together with the reply from the *Protocol Adapter* to the *Business Application* (which is defined above in detail).
 
-*TODO* add sequence diagram
+![Send Command](../command_control_Success.png)
 
 **Message Format**
 
-The following table provides an overview of the properties a client needs to set on a *Control* message.
+The following table provides an overview of the properties the *Business Application* needs to set on a *Control* message.
 
 | Name | Mandatory | Location | Type      | Description |
 | :--- | :-------: | :------- | :-------- | :---------- |
-| *message-id* | yes | *properties* | UTF-8 *string* | MUST contain the message ID used to match incoming control message with the response message containing result of execution. |
-| *device_id* | yes | *application-properties* | UTF-8 *string* | MUST contain the ID of the device the data in the payload has been reported by. |
-| *tenant_id* | yes | *application-properties* | UTF-8 *string* | MAY be set by the client to indicate which tenant the the device belongs to. |
-| *command* | no | *application-properties* | UTF-8 *string* | MUST contain the command to be executed by a device. |
-|*ttl* | no | *properties* | *int* | Indicates the timeout value (in milliseconds) indicating when server can assume that command has not been executed because of the lack of the response from the client. Default timeout is *60000* (1 minute). |
-|*durable* | no | *properties* | *boolean* | Indicates if the message should be persisted (*store and forward*) by the underlying message transport middleware. By default command&control messages are durable.  |
+| *message-id* | yes | *properties* | *string* | MUST contain the message ID used to match incoming control message with the response message containing result of execution. |
+| *subject* | yes | *properties* | *string* | MUST contain the command to be executed by a device. |
+| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
+| *ttl* | no | *properties* | *int* | Indicates the timeout value (in milliseconds) indicating when server can assume that command has not been executed because of the lack of the response from the client. |
```

Comment:
what is the *server* here? is this the *sender* of the command (i.e. the application)? Or is this Hono? If so, what happens once the ttl has expired?

Reply:
I agree that it is not clear. Since this is a standard property maybe reuse some of that description like:Duration in milliseconds for which the message is to be considered “live”. 
WDYT?